pyomo/core/staleflag.py

```diff
@@ -14,6 +14,15 @@ def __init__(self):
         self._current = 0
         self.mark_all_as_stale()
 
+    def stale_mapper(self, encode, value):
```

Comment:
I probably just missed it, but where does this get used?

Reply:
It's used in the `_GeneralVarData` / `_GeneralBooleanVarData` `__autoslot_mappers__`.